src/carousel/dto/create-carousel-slide.dto.ts

```diff
@@ -8,11 +8,9 @@ import {
   MaxLength,
 } from 'class-validator';
 import { CarouselLocation } from '@prisma/client';
+import { LanguageDto } from 'src/types/language.dto';
```

Comment:
After making a PR make sure to run the `yarn authorize-wallet` or some CLI command to make sure the CLI is still working